thespian/system/multiprocCommon.py

```diff
@@ -511,6 +510,28 @@ def shutdown_signal_detected(signum, frame):
         am.transport.interrupt_wait(signal_shutdown=True)
     return shutdown_signal_detected
 
+def get_min_log_level(logDefs:Dict[str, Any])->int:
```

Comment:
I appreciate the type annotations, but this syntax isn't valid for Python2.  While I'm ready to drop support for Python 2, I'd like to keep the support for 2.7 in this PR since this is a bugfix.

Reply:
fair point, removed annotations